omnigibson/prims/entity_prim.py

```diff
@@ -334,6 +320,20 @@ def _update_joint_limits(self):
             joint.lower_limit = joint.lower_limit * scale_along_axis
             joint.upper_limit = joint.upper_limit * scale_along_axis
 
+    @property
+    def _articulation_view(self):
+        if self._articulation_view_direct is None:
+            return None
+
+        # Validate that the articulation view is initialized and that if physics is running, the
+        # view is valid.
+        if og.sim.is_playing() and self.initialized:
+            assert self._articulation_view_direct.is_physics_handle_valid() and \
```

Comment:
have we checked edge cases for this? I recall some stop / playing cycles or adding objects at runtime can cause potentially issues with some physics views becoming invalid, IIRC

Reply:
Yes, I have checked those - I'm leaving the check in for now but I believe it's all covered (notice the new calls to update_handles I have added)